CONTRIBUTING.md

```diff
@@ -1,111 +1,167 @@
+1.  Sign in to your [github](github.com) account.
+1.  Navigate to the [Timescale documentation repo](github.com/timescale/docs).
+1.  Click the `Fork` button in the top-right corner, and select the account you
+    want to use.
 1.  Wait for GitHub to create your fork and redirect you.
-1.  Clone the repository to your local machine. To find this URL, click the green
-    `Code` button and copy the HTTPS URL:
+1.  Clone the repository to your local machine by clicking the green `Code`
+    button, copying the HTTPS URL, and running the following command at your
+    command prompt:
     ```bash
-    git clone https://github.com/<username>/docs.git
+    git clone <YOUR_FORK_URL>
     ```
 1.  List the current remote branches:
     ```bash
     git remote -v
     ```
     This command should list two remotes, both marked `origin`, like this:
     ```bash
-    origin  https://github.com/<username>/docs.git (fetch)
-    origin  https://github.com/<username>/docs.git (push)
+    origin  https://github.com/<YOUR_GITHUB_USERNAME>/docs.git (fetch)
+    origin  https://github.com/<YOUR_GITHUB_USERNAME>/docs.git (push)
     ```
-    The `origin` remotes are your own fork, and you can do whatever you want here without changing the upstream repository.
+    The `origin` remotes are your own fork. You can do whatever you want here
+    without changing the upstream repository.
 1.  Add the docs repo as an upstream:
     ```bash
     git remote add upstream https://github.com/timescale/docs.git
     ```
-1.  Check:
+1.  Check that the upstream repo was successfully added:
     ```bash
     git remote -v
     ```
-    This command should now have the same two `origin` remotes as before, plus two more labelled `upstream`, like this:
+    This command should now list the two `origin` remotes from before, plus two
+    more labelled `upstream`, like this:
     ```bash
-    origin  https://github.com/<username>/docs.git (fetch)
-    origin  https://github.com/<username>/docs.git (push)
+    origin  https://github.com/<YOUR_GITHUB_USERNAME>/docs.git (fetch)
+    origin  https://github.com/<YOUR_GITHUB_USERNAME>/docs.git (push)
     upstream  https://github.com/timescale/docs.git (fetch)
     upstream  https://github.com/timescale/docs.git (push)
     ```
 1.  Fetch the branches in the upstream repository:
     ```bash
     git fetch upstream
     ```
-1.  Merge the changes from the upstream `latest` branch, into your fork's
+1.  Merge the changes from the upstream `latest` branch into your fork's
     `latest` branch:
     ```bash
     git merge upstream/latest
     ```
-1.  Create a new branch for the work you want to do. Make sure you give it an
-    appropriate name, and include your username:
+1.  Create a new branch for the work you want to do. Give it a descriptive name
+    that includes your GitHub username. For example:
     ```bash
-    git checkout -b update-readme-username
+    git checkout -b update-readme-<USERNAME>
```

Comment:
I'm reliably informed that we're supposed to use `git switch` instead of `git checkout` these days. Might be worth updating.

Reply:
I thought `git switch` was still experimental, at least this is what the help page tells me:

`THIS COMMAND IS EXPERIMENTAL. THE BEHAVIOR MAY CHANGE.`

I'm fine with recommending it (it's what I use myself), but do we worry syntax will change?

Reply:
Ah, that's interesting, I didn't realise GH called it experimental! Let's leave it as-is for now.